.github/workflows/codecov_report_submit.yml

```
@@ -0,0 +1,39 @@
+name:  Codecov CI
+
+on:
+  push:
+    branches: [ trainers, abcd, ]
+  pull_request:
+    branches: [ trainers, abcd, ]
```

```suggestion
on:
  push:
    branches: trainers
  pull_request:
    branches: trainers
```
> @MrLYG Can we please update the status on this? Would this be ready to be merged?

Totally, yes.
And before the merge, I want to ask:
I remember you said to me that the `trainers` branch become the main branch we need to work on.
So, is it mean the `abcd` branch is deprecated? if it is, I will change the code here. That's mean we only focus on `abcd` trainers. Any push and pr submit to `abcd` will never trigger code-coverage to generate a report.

---

sounds good

---

> sounds good
Ready to merge.